nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/EncodeContent.java

```diff
@@ -112,11 +194,27 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
-        final boolean encode = context.getProperty(MODE).getValue().equalsIgnoreCase(ENCODE_MODE);
+        // Decide if we should fail on 0-length content
+        final Boolean failOnZeroLengthContent = context.getProperty(FAIL_ON_ZERO_LENGTH_CONTENT).asBoolean();
```

Comment:
```suggestion
        final boolean failOnZeroLengthContent = context.getProperty(FAIL_ON_ZERO_LENGTH_CONTENT).asBoolean();
```

Reply:
`FAIL_ON_ZERO_LENGTH_CONTENT` is removed as per @exceptionfactory 's suggestion